### PR TITLE
Pull Request: Security

### PR DESCRIPTION
**Pull request by katsiaryne**
# Related Tasks
* Rename driver_id filed
* Add security

### CODE REVIEW

File: src/main/kotlin/com/software/modsen/driverservice/dto/request/InitDriverRequest.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.software.modsen.driverservice.dto.request
+
+data class InitDriverRequest (
+    val name: String,
+    val email: String,
```
Comment (psidarovich) [inline src/main/kotlin/com/software/modsen/driverservice/dto/request/InitDriverRequest.kt:13]: Minor: What about validation on email?

File: src/main/kotlin/com/software/modsen/driverservice/dto/response/DriverResponse.kt [hunk 2]
```diff
@@ -3,7 +3,7 @@ package com.software.modsen.driverservice.dto.response
 import com.software.modsen.driverservice.model.Sex
 
 data class DriverResponse(
-    val driverId: Long?,
+    val id: Long?,
```
Comment (psidarovich) [inline src/main/kotlin/com/software/modsen/driverservice/dto/response/DriverResponse.kt]: Are you sure, that your id can be null in response?

**Review by psidarovich — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/software/modsen/driverservice/dto/request/InitDriverRequest.kt (hunk 1 above)
Comment (katsiaryne) [reply src/main/kotlin/com/software/modsen/driverservice/dto/request/InitDriverRequest.kt:13]: Fix: add field validation

File: src/main/kotlin/com/software/modsen/driverservice/dto/response/DriverResponse.kt (hunk 2 above)
Comment (katsiaryne) [reply src/main/kotlin/com/software/modsen/driverservice/dto/response/DriverResponse.kt]: Fix:  Id cannot be null.

After getting an object from the database, it always has an id, so I decided to use an <id!!> in mapper

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -32,8 +32,8 @@ spring:
           issuer-uri: http://localhost:8100/realms/cab-aggregator-realm
           jwk-set-uri: http://localhost:8100/realms/cab-aggregator-realm/protocol/openid-connect/certs
 eureka:
-  instance:
-    hostname: driver-service
+ # instance:
+   # hostname: driver-service
```
Comment (psidarovich) [inline src/main/resources/application.yml]: Why did you comment this code?

**Review by psidarovich — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 3 above)
Comment (katsiaryne) [reply src/main/resources/application.yml]: Fixed

**Review by psidarovich — APPROVED** [review]

**Pull request merged**